content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -2,192 +2,206 @@
+   organization called "ServiceNow", and ensure that it has the
+   organization-level ["Manage Workspaces"
+   permission](../../users-teams-organizations/permissions.html#manage-workspaces).
+   You do not need to add any users to this team.
+1. On the "ServiceNow" team's settings page, generate a [team API
+   token](../../users-teams-organizations/api-tokens.html#team-api-tokens).
+   **Save the Team API Token for later.**
+1. If you haven't yet done so, [connect a VCS provider](../../vcs/index.html) to
+   this Terraform organization. This provider will include one or more
+   repositories with Terraform configuration. Any repository that could be
+   connected to a manually-created Terraform Cloud workspace can also be
+   used as a workspace template in the ServiceNow integration.
+1. On your organization's VCS provider settings page (Settings > VCS Providers),
+   find the "OAuth Token ID" for the VCS provider or providers you will be using
+   with the ServiceNow integration. Terraform Cloud uses the OAuth Token ID to
+   identify and authorize the VCS provider.
+   **Save the OAuth Token ID for later.**
+1. Identify one or more VCS repositories in the VCS provider containing
+   Terraform configuration that will be provisioned by the ServiceNow Terraform
+   integration.
+1. Provide the following information to the ServiceNow Admin.
+  1. The Organization Name
+  1. The Team API Token
+  1. The hostname of your Terraform Enterprise instance, or of Terraform Cloud.
+     The hostname of Terraform Cloud is `app.terraform.io`.
+  1. The OAuth Token ID(s) and identifier(s) for the VCS repository or
+     repositories containing Terraform configuration that will be used by the
+     integration.
+  1. Any Terraform or environment variables required by the configuration in the
+     given VCS repositories.
+
+-> **Note:** Repository identifiers are determined by your VCS provider; they
+typically use a format like `<ORGANIZATION>/<REPO_NAME>` or
+`<PROJECT_KEY>/<REPO_NAME>`. Azure DevOps repositories use the format
+`<ORGANIZATION>/<PROJECT>/_git/<REPO_NAME>`. A GitHub repository hosted at
+`https://github.com/exampleorg/examplerepo/` would have the repository
+identifier `exampleorg/examplerepo`.
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## Installing the ServiceNow Integration
-
-#### Enable Polling Workers (Recommended)
+For instance, if you are configuring this integration for your company, `Example
+Corp`, using two GitHub repositories, you would share values like the following
+with the ServiceNow Admin.
 
-The integration includes 3 ServiceNow Scheduled Flows to poll the Terraform Enterprise API using ServiceNow Outbound HTTP REST requests. By default, all flows schedules are set to 5 minutes. These can be customized inside the ServiceNow Server Studio:
+![example: Configuration Values](./images/example-configuration-values.png "Example Configuration Values")
```

Comment:
This example content should be provided as a textual code block, not as an image of text. (Accessibility.)

Reply:
Oh, that's a really good point. Fixed!